torch/csrc/utils/python_numbers.h

```diff
@@ -41,10 +41,15 @@ inline PyObject* THPUtils_packDoubleAsInt(double value) {
 }
 
 inline bool THPUtils_checkLong(PyObject* obj) {
+  bool is_numpy_int = false;
+#ifdef USE_NUMPY
+  is_numpy_int = torch::utils::is_numpy_int(obj);
```

Comment:
could we just do
```
if(torch::utils::is_numpy_int(obj)) [
    return true
}
```
instead of adding the "|| is_numpy_int" condition below?

Reply:
yes that'll probably be clearer